app/[locale]/signin/components/signInForm.tsx

```diff
@@ -0,0 +1,117 @@
+'use client';
+
+import { cn } from '@/lib/utils';
+import { Button } from '@/components/ui/button';
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from '@/components/ui/card';
+import { Input } from '@/components/ui/input';
+import { Label } from '@/components/ui/label';
+import { ComponentProps, useState } from 'react';
+import { useTranslations } from 'next-intl';
+
+import { getAuthSchema } from '@/lib/schemas/auth';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { z } from 'zod';
+import { useForm } from 'react-hook-form';
+import { signIn } from 'next-auth/react';
+import { useRouter } from 'next/navigation';

```

Comment:
Используй враппер от next-intl. Все доступные врапперы лежат в src/i18n/navigation.ts
```suggestion
import { useRouter } from '@i18n';
```

Reply:
исправлено